### PR TITLE
#74: UncheckedScalar, UncheckedText, and UncheckedBytes

### PR DESCRIPTION
**Pull request by fabriciofx**

### CODE REVIEW

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/func/UncheckedScalar.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+/**
+ * Scalar that doesn't throw checked {@link Exception}.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Fabricio Cabral (fabriciofx@gmail.com)
+ * @version $Id$
+ * @param <T> Type of result
+ * @since 0.2
```
Comment (fabriciofx) [inline src/main/java/org/cactoos/func/UncheckedScalar.java]: @g4s8 Ops, my mistake. I'll change it. Thanks!

File: src/main/java/org/cactoos/func/UncheckedScalar.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    public UncheckedScalar(final Scalar<T> scalar) {
+        this.scalar = scalar;
+    }
+
+    @Override
+    public T asValue() {
+        try {
+            return this.scalar.asValue();
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
```
Comment (fabriciofx) [inline src/main/java/org/cactoos/func/UncheckedScalar.java]: @g4s8 Well, I just followed the same approach was used [here](https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/func/UncheckedFunc.java). 😄 
But you're right. `IllegalStateException` isn't the better exception in this case. Fell free to suggest another one.

Comment (g4s8) [inline src/main/java/org/cactoos/func/UncheckedScalar.java]: @fabriciofx maybe `java.io.UncheckedIOException` would be better? https://docs.oracle.com/javase/8/docs/api/java/io/UncheckedIOException.html

Comment (fabriciofx) [inline src/main/java/org/cactoos/func/UncheckedScalar.java]: @g4s8 I think it very good. Thanks!

**Comment by yegor256** [comment]
@rultor try to merge

**Pull request merged**